### PR TITLE
Add(-): basic landing page. Integration with Material UI

### PR DESCRIPTION
**Pull request by evinracher**
- Integrate Material UI (client-site rendering with caching, pending: investigate SSR integration)
- Improve landing page. Add basic layout and AppBar

Further work: Improve landing page and add dashboard section (I will do this in further PRs)

### CODE REVIEW

File: src/components/layout/layout.tsx [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import { Container } from '@mui/material';
+import { ReactNode } from 'react';
+import { Navbar } from '../navbar/navbar';
```
Comment (nietoga) [inline src/components/layout/layout.tsx:3]: Just a note for the future. We gotta discuss if it's better using relative or absolute paths for imports.

File: src/pages/_app.tsx [hunk 2]
```diff
@@ -2,19 +2,39 @@ import { Meta } from '@nietoga/nietoga-com/components/page';
...
                 name="viewport"
                 content="width=device-width, initial-scale=1"
             />
             <Head>
                 <link rel="icon" href="/favicon.ico" />
             </Head>
-            <Component {...pageProps} />
-        </>
+            <ThemeProvider theme={theme}>
+                {/* CssBaseline kickstart an elegant, consistent, and simple baseline to build upon. */}
```
Comment (nietoga) [inline src/pages/_app.tsx:33]: This comment:

image

File: src/themes/theme.js [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+import { red } from '@mui/material/colors';
+
+export const roboto = Roboto({
+    weight: ['300', '400', '500', '700'],
+    subsets: ['latin'],
+    display: 'swap',
+    fallback: ['Helvetica', 'Arial', 'sans-serif'],
+});
+
+// Create a theme instance.
```
Comment (nietoga) [inline src/themes/theme.js:12]: Unnecessary comment

File: src/utils/createEmotionCache.ts [hunk 4]
```diff
@@ -0,0 +1,19 @@
+import createCache from '@emotion/cache';
+
+const isBrowser = typeof document !== 'undefined';
+
+// On the client side, Create a meta tag at the top of the <head> and set it as insertionPoint.
+// This assures that MUI styles are loaded first.
+// It allows developers to easily override MUI styles with other styling solutions, like CSS modules.
```
Comment (nietoga) [inline src/utils/createEmotionCache.ts:7]: Useless comment.

File: src/pages/_app.tsx [hunk 5]
```diff
@@ -2,19 +2,39 @@ import { Meta } from '@nietoga/nietoga-com/components/page';
...
 import type { AppProps } from 'next/app';
 import Head from 'next/head';
+import createEmotionCache from '../utils/createEmotionCache';
+import { CacheProvider, EmotionCache } from '@emotion/react';
+import { ThemeProvider } from '@mui/material/styles';
+import CssBaseline from '@mui/material/CssBaseline';
+import { theme } from '../themes/theme';
 
-const App = ({ Component, pageProps }: AppProps) => {
+// Client-side cache, shared for the whole session of the user in the browser.
```
Comment (nietoga) [inline src/pages/_app.tsx:11]: Useless comment

**Review by nietoga — APPROVED** [review]

File: src/pages/_app.tsx (hunk 5 above)
Comment (evinracher) [reply src/pages/_app.tsx]: Removed

File: src/pages/_app.tsx (hunk 2 above)
Comment (evinracher) [reply src/pages/_app.tsx]: Removed

File: src/themes/theme.js (hunk 3 above)
Comment (evinracher) [reply src/themes/theme.js]: Removed

File: src/utils/createEmotionCache.ts (hunk 4 above)
Comment (evinracher) [reply src/utils/createEmotionCache.ts]: Remove

**Pull request merged**